client/src/scripts/managers/inputManager.ts

```diff
@@ -342,6 +343,50 @@ export class InputManager {
                 shootOnRelease = false;
             });
         }
+        const ticker = new PIXI.Ticker();
+        ticker.stop();
+        ticker.add(() => {
+            const gamepads = navigator.getGamepads();
+            if (gamepads[0]) {
+                const leftJoystickMoving = gamepads[0].axes[0] !== 0 || gamepads[0].axes[1] !== 0;
+                const rightJoystickMoving = gamepads[0].axes[2] !== 0 || gamepads[0].axes[3] !== 0;
```

Comment:
rest in peace stick drift controllers

Reply:
Perhaps there could be a sensitivity setting to adjust when the JoystickMoving vars are considered true

Reply:
Resolved, both joysticks now have a sensitivity slider